definitions/checks/pulpcore/no_running_tasks.rb

```diff
@@ -0,0 +1,32 @@
+module Checks::Pulpcore
+  class NoRunningTasks < ForemanMaintain::Check
+    metadata do
+      for_feature :pulpcore
+      description 'Check for running pulpcore tasks'
```

Comment:
Should we switch to calling it Pulp?

Reply:
We're using "Pulpcore"/"pulpcore" everywhere else right now in messages. So "maybe, but then we should do it consistently"?

Reply:
Fair. Let's track that in an issue?

Reply:
https://github.com/theforeman/foreman_maintain/issues/902